test/comment.ts

```diff
@@ -0,0 +1,99 @@
+// Needed due to chai's should.exist
+/* tslint:disable: no-unused-expression */
+import * as chai from 'chai';
+import * as ChaiAsPromised from 'chai-as-promised';
+import * as _ from 'lodash';
+import 'mocha';
+import { Front, FrontComment, FrontCommentMentions } from '../lib/index';
+import { getKeeper } from './keeper';
+
+chai.use(ChaiAsPromised);
+chai.should();
+
+require('./message');
```

Comment:
This is to ensure they run in the correct order in Mocha, else it'll try them as it finds them. I need some of the particular tests (like the message tests) to run before the comment and conversation tests, for example.